### DISCUSSION TITLE
Remove Yellow Highlight after clicking a link

### DISCUSSION THREAD

**Discussion by OTJArchitects** [sumatrapdfreader/sumatrapdf#4085] [Q&A] [answered]
I've been enjoying using SumatraPDF for pitches.

I'm wondering if there is a way to disable the highlighter yellow covering links. In our presentations we usually make links pictures, and when we ctlr+w out of the clicked presentation, the host presentation has a yellow highlight filter over the clicked link. This is only making client pitches slightly unsightly but would be awesome if there was a way to turn it off. I was in the advanced settings and say "highlighter color" or something along those lines, but I don't think that does anything to the link clicking highlight color.

Thanks!

**Comment by GitHubRulesOK** [comment, marked as answer]
That is the highlighted "selection" and its colour is controlled by 
`SelectionColor = #f5fc0c`

You can set that to a different color such as `#ffffff` but then you will not see other selections apart from the selection box !

**Reply by OTJArchitects** [reply]
This actually made the box white which added a white highlight. Turns out, my team members are rather happy with the white highlight as it points to where "they" just were while not being "in your face." Thank you for the response! Maybe, eventually having an option to remove the highlight color and outline all together for specific usage-cases, but it doesn't seem necessary. Thanks, again!
